packages/shared/src/utils/config/config.js

```diff
@@ -45,6 +53,10 @@ export const getAppId = () => {
     ) {
         window.localStorage.removeItem('config.default_app_id');
         app_id = isBot() ? 19112 : 16303; // it's being used in endpoint chrome extension - please do not remove
```

Comment:
can we move this line to else block at line no #59

Reply:
Isn't line 59 reserved for staging? :thinking: